docs/blog/articles/2021-10-11-KServe-0.7-release.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "The KServe 0.7 release ensures a smooth transition from KFserving to KServe"
+date: 2020-10-11
+description: "KServe 0.7 Release Blog Post"
+type: "blog"
+---
+
+### Authors
+ **Dan Sun**,**Animesh Singh**,**Vedant Padwal** on behalf of the KServe Community
+
+### **What's Changed?**
+
+- #### [Migration job for existing KFServing installation](https://kserve.github.io/website/admin/migration)
+    This helps to migrate existing inference services running in the cluster from KFServing to KServe without downtime. 
+    The script installs `KServe` from 0.7 release and deletes your `KFServing` installation after migrating the inference services from `serving.kubeflow.org` to `serving.kserve.io`.
+<br>
+- `InferenceService` API group is changed from `serving.kubeflow.org` to `serving.kserve.io` [#1826](https://github.com/kserve/kserve/issues/1826).
+- Python SDK name is changed from `kfserving` to `kserve` [#1827](https://github.com/kserve/kserve/issues/1827).
```

Comment:
Can you help provide the link to the pypi package https://pypi.org/project/kserve ?